x-pack/test/screenshot_creation/apps/ml_docs/anomaly_detection/geographic_data.ts

```diff
@@ -223,7 +227,7 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
     });
 
     // the job stopped to produce an anomaly, needs investigation
```

Comment:
Nit: this comment can be removed

Reply:
Fixed in 15160632eb52222b5335ec5447e2baec47c80f2c